phpBB/common.php

```diff
@@ -134,6 +134,8 @@
 
 require($phpbb_root_path . 'includes/compatibility_globals.' . $phpEx);
 
+register_compatibility_globals();
```

Comment:
I have removed the automatic call as it causes issue when running the UI tests (when running the full test suite, register_compatibility_globals isn't caleld in the UI tests installer because the file is already included... basics of file inclusion with side effect actually)